src/Api/Learn.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace HappyrMatch\ApiClient\Api;
+
+use HappyrMatch\ApiClient\Exception;
+use HappyrMatch\ApiClient\Model\Accepted;
+use Psr\Http\Message\ResponseInterface;
+use Webmozart\Assert\Assert;
+
+/**
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+class Learn extends HttpApi
+{
+    /**
+     * @throws Exception
+     *
+     * @return ResponseInterface|Accepted
+     */
+    public function create(string $candidate, string $role, string $learnName, string $category, ?int $score = null)
+    {
+        Assert::notEmpty($candidate, 'Candidate cannot be empty');
+        Assert::notEmpty($role, 'Role cannot be empty');
+        Assert::notEmpty($learnName, 'Learn name cannot be empty');
+        Assert::notEmpty($category, 'Category cannot be empty');
+        Assert::oneOf($category, [
```

Comment:
We should let the API handle this instead.

I suggest moving these categories to the doc block instead.

Reply:
I'll remove them then, since we have the allowed values in the docs

Reply:
Yeah, but if you put them in the doc block you can get IDE autocompletion.